runtime/docker/firewall/nftables/client.go

```diff
@@ -45,6 +45,15 @@ func NewNftablesClient(bridgeName string) (*NftablesClient, error) {
 		bridgeName: bridgeName,
 	}
 
+	chains, err := nftC.getChains(definitions.DockerFWUserChain)
+	if err != nil {
+		return nil, err
+	}
+	if len(chains) == 0 {
```

Comment:
Can we check for docker user chain and not just some chains?

Reply:
Thats what we do. if you look closely you will notice `definitions.DockerFWUserChain` as the parameter to `getChains(...)`.